packages/strapi-design-system/src/DateTimePicker/__tests__/DateTimePicker.e2e.js

```diff
@@ -0,0 +1,137 @@
+const { injectAxe } = require('axe-playwright');
+
+const { test, expect } = require('@playwright/test');
+
+test.describe.parallel('DateTimePicker', () => {
+  test.describe('without initial data', () => {
+    test.beforeEach(async ({ page }) => {
+      // This is the URL of the Storybook Iframe
+      await page.goto('/iframe.html?id=design-system-components-datetimepicker--base&viewMode=story');
+      await injectAxe(page);
+    });
+    test('change the date and see if also the time will be changed when both are empty', async ({
+      page,
+      browserName,
+    }) => {
+      test.skip(browserName === 'webkit', 'Still working on it');
+      await page.focus('[aria-labelledby="datetime-label"] > div > div:nth-child(2) > div > div');
+
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Enter');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Tab'); // move to the Su column
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('ArrowDown'); // move to the third Sunday of the month
+      await page.keyboard.press('Enter'); // to select the date
+      // check if the time is selected with 00:00
+      const timePickerValue = page.locator('span:has-text("00:00")');
+      expect(await timePickerValue.innerText()).toBe('00:00');
+    });
+    test('change the time and see if also the date will be changed when both are empty', async ({
+      page,
+      browserName,
+    }) => {
+      test.skip(browserName === 'webkit', 'Still working on it');
+      await page.focus('[aria-labelledby="datetime-label"] > div > div:nth-child(2) > div');
+
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Tab');
+      await page.keyboard.press('Enter');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('ArrowDown');
+      await page.keyboard.press('Enter'); // to select the time
```

Comment:
Same question about all the keyboard press and same for the next tests.

Reply:
Basically, I have implemented the navigation with the keyboard because I have found also some bugs inside the Component code with it, but if you are not ok with it I can refactor the code using a simple click event

I had to use the skip because in webkit we have the tests stuck, we have the same strange behavior and used skip in another test inside the DS repo

Reply:
done